lib/core/versions/latest/ErrorCode.ts

```diff
@@ -122,8 +122,8 @@ export default {
   MultihashUnsupportedHashAlgorithm: 'multihash_unsupported_hash_algorithm',
   OperationCreateInvalidDidDocument: 'operation_create_invalid_did_document',
   OperationCountLessThanZero: 'operation_count_less_than_zero',
-  OperationCreatePayloadHasMissingOrInvalidNextRecoveryCommitment: 'operation_create_payload_has_missing_or_invalid_next_recovery_commitment',
-  OperationCreatePayloadHasMissingOrInvalidNextUpdateCommitment:  'operation_create_payload_has_missing_or_invalid_next_update_commitment',
+  OperationCreatePayloadHasMissingOrInvalidNextRecoveryCommitment: 'operation_create_payload_has_missing_or_invalid_next_recoveryCommitment',
+  OperationCreatePayloadHasMissingOrInvalidNextUpdateCommitment:  'operation_create_payload_has_missing_or_invalid_next_updateCommitment',
```

Comment:
Find and replace error. I realized this is painful, I am convinced that property and value being the same casing now trumps the "friendliness" to the reader of these messages, just haven't gotten around to write a script to change them all over.

Reply:
https://github.com/decentralized-identity/sidetree/pull/881/commits/5f3194cd745beabfdd33e4ed3717cd5d482174e7